internal/db/db.go

```diff
@@ -86,6 +79,7 @@ func (d *DB) Close(ctx context.Context) error {
 // answer is no, but there are occasions when it's necessary.  See the sql.DB
 // docs for more information.
 func Open(dbType DbType, connectionUrl string, opt ...Option) (*DB, error) {
+	const op = "db.Open"
 	var dialect gorm.Dialector
```

Comment:
this looks to be the reason that gorm is still being imported into this file. This is passed to `dbw.OpenWith`, but it looks like there is a `dbw.Dialector` that is a thin wrapper ofr `gorm.Dialector`, so I think this should be changed to `var dialect dbw.Dialector` and that will remove `gorm` from the imports.

Reply:
ty!